v0/pipeline/steps/common/artifact-signing.yaml

```diff
@@ -0,0 +1,35 @@
+title: Signing
+type: object
+properties:
+  type:
+    type: string
+    enum:
+      - cosign
+  description:
+    desc: This is the description for Signing
+  spec:
+    type: object
+$schema: http://json-schema.org/draft-07/schema#
+allOf:
+  - if:
+      properties:
+        type:
+          const: cosign
+    then:
+      properties:
+        spec:
+          type: object
+          oneOf:
+            - $ref: cosign-signing.yaml
```

Comment:
Should we keep this same everywhere since we are reusing pretty much everything.

Reply:
kept it same